pkg/actors/engine/engine.go

```diff
@@ -74,6 +76,12 @@ type engine struct {
 
 	lock  *fifo.Mutex
 	clock clock.Clock
+
+	actorCallOpts actorCallOptions
```

Comment:
We can pre build the gRPC options on `New`.

```suggestion
	callOptions []grpc.CallOption
```

Reply:
Changed